project/ws/app/src/lib/routes/home/components/request-list/assign-list-popup/assign-list-popup.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnInit } from '@angular/core'
+import { FormBuilder, FormControl, FormGroup } from '@angular/forms'
+
+@Component({
+  selector: 'ws-app-assign-list-popup',
+  templateUrl: './assign-list-popup.component.html',
+  styleUrls: ['./assign-list-popup.component.scss'],
+})
+export class AssignListPopupComponent implements OnInit {
+  requestForm!: FormGroup
+  providers = [
+    { orgName: 'Microsoft',
+     details: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec suscipit orci in ultricies aliq.',
```

Comment:
is dummy data required?

Reply:
api is not provided yet ..will mention with qa people